client.go

```diff
@@ -68,7 +68,7 @@ func (c *Client) SubmitPFB(ctx context.Context, namespaceID [8]byte, data []byte
 	return &res, nil
 }
 
-func (c *Client) NamespacedShares(ctx context.Context, namespaceID [8]byte, height uint64) ([][]byte, error) {
+func (c *Client) NamespacedShares(ctx context.Context, namespaceID [28]byte, height uint64) ([][]byte, error) {
```

Comment:
We should introduce a type/alias for `[28]byte`.

Reply:
Added.